scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultIllumina.java

```diff
@@ -98,7 +98,7 @@ public final class DefaultIllumina extends RunProcessor {
   private static final XPathExpression WORKFLOW_TYPE;
   private static final XPathExpression SBS_CONSUMABLE_VERSION;
   private static final XPathExpression START_TIME_XPATH;
-  private static final XPathExpression READ_2_REVERSE_COMPLIMENT;
+  private static final XPathExpression I5_REVERSE_COMPLIMENT;
```

Comment:
Spelling is `complement` in this case, but this isn't visible to users so doesn't actually matter

Reply:
Fixed anyway. Thanks for the English lesson! 😅